docs_src/design/TOC.md

```diff
@@ -15,6 +15,7 @@
 | [System Events for Devices](./ucr/System-Events-for-Devices.md)                     | Use Case for System Events for Device add/update/delete             |
 | [Microservice Authentication](./ucr/Microservice-Authentication.md)                 | Use Case for Microservice Authentication                            |
 | [URIs for files](.//ucr/URIs-for-Files.md)                                          | Use Case for loading service files from URIs                        |
+| [Core Data Retention and Persistent Cap](./ucr/Core-Data-Retention.md)              | Use Case for capping readings in Core Data |
```

Comment:
UCR are in alphabetical order. Please move up to appropriate position.
This can be addressed when changing the USR to approved.